doc/classes/RichTextLabel.xml

```diff
@@ -285,7 +285,7 @@
 	</methods>
 	<members>
 		<member name="bbcode_enabled" type="bool" setter="set_use_bbcode" getter="is_using_bbcode" default="false">
-			If [code]true[/code], the label uses BBCode formatting.
+			If [code]true[/code], the label uses BBCode formatting. If anything other than [code]set_bbcode[/code] is used to alter the [RichTextLabel]s text this will be reset to [code]false[/code].
```

Comment:
Style fix:
```suggestion
			If [code]true[/code], the label uses BBCode formatting.
			[b]Note:[/b] Using any other method than [method set_bbcode] to alter the [RichTextLabel]'s BBCode-enabled text will reset this to [code]false[/code].
```

But more importantly, I'm not sure this is true. What of `append_bbcode`, and all the `push_*` methods?